File: app/src/main/java/com/faa1192/weatherforecast/Countries/Country.java
package com.faa1192.weatherforecast.Countries;

import android.content.Context;

import com.faa1192.weatherforecast.R;

import java.util.HashMap;
import java.util.Map;

public class Country {
    private final Map<String, Integer> map = new HashMap<>();
    private final Context context;

    public Country(Context context) {
        this.context = context;
        map.put("RU", R.string.RU);
        map.put("UA", R.string.UA);
    }

    public String getName(String s) {
        Integer i = map.get(s);
        if (i == null)
            return s;
        return context.getResources().getString(i);
    }

}

File: app/src/main/java/com/faa1192/weatherforecast/countries/Country.kt
package com.faa1192.weatherforecast.countries

import android.content.Context
import com.faa1192.weatherforecast.R

class Country(private val context: Context) {
    private val map: MutableMap<String, Int> = HashMap()
    fun getName(s: String): String {
        val i = map[s] ?: return s
        return context.resources.getString(i)
    }

    init {
        map["RU"] = R.string.RU
        map["UA"] = R.string.UA
    }
}